docs/using-flatpak.rst

```diff
@@ -62,17 +78,26 @@ The same principle applies to repositories - repositories that have been
 added system-wide are available to all users, whereas per-user repositories
 can only be used by a particular user.
 
-Flatpak commands are run system-wide by default. If you are installing
-applications for day-to-day usage, it is recommended to stick with this
-default behavior.
+.. Flatpak commands are run system-wide by default. If you are installing
+.. applications for day-to-day usage, it is recommended to stick with this
+.. default behavior.
+
+Flatpak commands are run system-wide by default. If you are installing 
+applications for day-to-day usage, it is recommended to use ``--user`` if 
```

Comment:
This is not recommended. `--user` is only really recommended if you lack permissions to do it system wide.

Reply:
It's a simpler process for a user to use `--user` by default, rather than letting everybody on the system have access to it, which, as you said, would also require user authentication to modify the required directories on the root partition. I also haven't seen a case of an app breaking due to a flatpak being installed per-user, rather than on the system (which was mentioned before, when `--user` was recommended to be the default)